### DISCUSSION TITLE
Image für 7430 nicht kompilierbar

### DISCUSSION THREAD

**Discussion by karstengit** [Freetz-NG/freetz-ng#658] [Allgemein] [answered]
Die Beschreibung ist unter Bug https://github.com/Freetz-NG/freetz-ng/issues/655 zu finden.

Es ist leider nicht nachvollziehbar warum die Kernel-Module angemeckert werden und woher ein sha1_generic.ko gezaubert werden soll?

**Comment by karstengit** [comment, marked as answer]
Nun konnte ein Image erstellt werden.

Das Verfahren sieht so aus das zunächst ohne weitere Hinzufügung für ein Modell kompiliert wird.
Wenn das funktioniert können schrittweise Packages hinzugefügt und nachfolgend getestet werden.

Welche Einstellungen zu dem oben aufgeführten Fehler geführt haben ist leider nicht nachvollziehbar.

**Reply by fda77** [reply]
> Nun konnte ein Image erstellt werden. 

Klingt gut

**Comment by karstengit** [comment]
Der beschriebene Fehler mit den fehlenden Kernel-Modulen tritt ganz klar auf, wenn PPP unter den Weboberflächen aktiviert wird.

![ppp](https://user-images.githubusercontent.com/5941550/205504116-62b57747-90a4-4363-a347-580020170149.jpg)

Wenn das Mal kein Bug ist ...
Ist halt ein neues Feature welches noch nicht richtig funktioniert.
